roles/edpm_bootstrap/tasks/swap.yml

```diff
@@ -16,6 +16,7 @@
 
 - name: Configure swap file
   when:
+    - not ansible_local.bootc
```

Comment:
we requrie swap for  all computre nodes, how will that requirement be met for bootc?